src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx

```diff
@@ -70,6 +70,20 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
     };
   }
 
+  public async getBountyEth(values: IFormValues): Promise<string> {
+    const currentAction = this.state.currentAction;
+    var ethToSend: number = 0;
+
+    for (var field of currentAction.getFields()) {
+      if (field.name === '_depositAmount' || field.name === '_amount') {
+        ethToSend += parseInt(field.callValue(values[field.name]))/1000000000;
```

Comment:
Is dividing by 10^9 correct here? 
If the original value is in Eth and the resul tin Wei,I'd expect this to be something like fieldValue * 10^18
We have a helper function called `toWei` in `src/util.ts`

Reply:
Yes. For some reason, this value always adds 1e18, regardless of the amount of decimals specified in the scheme JSON. So if 18 decimals were specified, the value would be 1e36 Wei. I specified 9 decimals, to make the field Gwei, as so to enable ETH contributions of fractions of 1 ETH, so to get the correct Wei amount, I divided by 1e9, for the 9 decimals. Without this change, the transaction had tried to transfer 1e9 times that what the user requested.